src/Symfony/Component/HttpFoundation/StreamedResponse.php

```diff
@@ -68,14 +68,9 @@ public static function create($callback = null, $status = 200, $headers = array(
      * Sets the PHP callback associated with this Response.
      *
      * @param callable $callback A valid PHP callback
-     *
-     * @throws \LogicException
      */
-    public function setCallback($callback)
+    public function setCallback(callable $callback)
```

Comment:
This is missing the typehint in the constructor from https://github.com/symfony/symfony/pull/16125/files#diff-f4e1f29f5c748cae819b5bce4096d66dR39